typescript/best-practice-web/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "typescript",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "wdio run ./wdio.conf.ts",
```

Comment:
Try to keep all the scripts the same, like running locally, on Sauce and so on so the script structure is the same for each folder, see [here](https://github.com/saucelabs-training/demo-js/blob/main/webdriverio/webdriver/best-practices/package.json#L7) for an example

Reply:
@wswebcreation I didn't get it, you like the dot notation? `test.saucelabs.eu`